pxtrunner/runner.ts

```diff
@@ -384,20 +384,18 @@ namespace pxt.runner {
     }
 
     export let editorLanguageMode = LanguageMode.Blocks;
-    export let editorLocale = "en";
 
-    export function setEditorContextAsync(mode: LanguageMode, locale: string) {
+    export function setEditorContextAsync(mode: LanguageMode, localeInfo: string) {
         editorLanguageMode = mode;
-        if (locale != editorLocale) {
+        if (localeInfo != pxt.Util.localeInfo()) {
```

Comment:
`pxt.Util.localeInfo()` looks different than `editorLocale` (`editorLocale` can't ever have the `live-` prefix). Should this be `pxt.Util.userLanguage()` instead?

Reply:
We actually pass the localeInfo to the side docs from the editor and this function assumes that live- might be present. I'll take a peek at the calls.

Reply:
Reviewed the calls and "localeInfo" is passed along.